src/dpsvalidationservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/dpsvalidationservice/GetValidOpenDFCMCase.java

```diff
@@ -37,7 +37,7 @@
     @RequestMapping(value = "/getValidOpenDFCMCase",
             produces = {"application/xml"},
             method = RequestMethod.GET)
-    @ApiOperation(value = "Generates application callback endpoints for DPS Validation Service", notes = "", tags = {"DPSValidationService"})
+    @ApiOperation(value = "Generates application callback endpoints for Driver Fitness Case Management Validation Service", notes = "", tags = {"DPSValidationService"})
```

Comment:
Please remove the "Generates application callback endpoints for". Sorry if I wasn't clear. This is not a callback. Thx.

Reply:
removed" Generates application callback endpoints for"

Reply:
done